extensions/amp-video/0.1/test-e2e/test-amp-video.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+describes.endtoend(
+  'amp-video autoplay with control',
+  {
+    testUrl: 'http://localhost:8000/test/fixtures/e2e/amp-video/autoplay.html',
+    environments: 'amp4ads-preset',
```

Comment:
This test is simple enough that we can turn it on for all of our environments (it shouldn't break). The naming of this file does make it the generic amp-video e2e test. We can either rename this test so that it is specific for ads, or we would probably expect that future e2e tests are added here and run in all environments.

Reply:
I wanted to. Not sure why but PWA is not cooperating for now. I believe it was something about the test controller (have to scroll the page to trigger certain events) that could be fixed later.